native_client/deepspeech.cc

```diff
@@ -241,6 +173,105 @@ Model::infer(float* aMfcc, int aNFrames, int aFrameLen)
 }
 
 char*
+Model::infer(float* aMfcc, int aNFrames, int aFrameLen)
+{
+  const int batch_size = BATCH_SIZE;
+  const int timesteps = aNFrames;
+  const size_t num_classes = mPriv->alphabet->GetSize() + 1; // +1 for blank
+
+  const int frameSize = mPriv->ncep + (2 * mPriv->ncep * mPriv->ncontext);
+
+  float*** input_data_mat = (float***)calloc(timesteps, sizeof(float**));
```

Comment:
`input_data_mat` is never freed. You should use `unique_ptr` or free it at the end of `decode`.

Reply:
Yep, good catch.

Reply:
Added `free()` calls at the end of `decode`.